src/Vts.Scripting/MonteCarlo/Demo06pMCPostProcessor.cs

```diff
@@ -0,0 +1,121 @@
+﻿namespace Vts.Scripting.MonteCarlo;
+
+/// <summary>
+/// Class using the Vts.dll library to demonstrate using the Perturbation Monte Carlo post-processor
+/// </summary>
+internal class Demo06PmcPostProcessor : IDemoScript
+{
+    /// <summary>
+    /// Sample script to demonstrate this class' stated purpose
+    /// </summary>
+    public static void RunDemo(bool showPlots = true)
+    {
+        // Example 06: run a Monte Carlo simulation with Perturbation Monte Carlo (pMC) post-processing enabled.
+        // First run a simulation, then post-process the generated database with pMC, varying optical properties post-simulation
+        // Notes:
+        //    - default source is a point source beam normally incident at the origin 
+
+        // create a SimulationInput object to define the simulation
+        var tissueInput = new MultiLayerTissueInput
+        {
+            Regions = new ITissueRegion[]
+                {
+                    new LayerTissueRegion(
+                        zRange: new(double.NegativeInfinity, 0),         // air "z" range
+                        op: new(mua: 0.0, musp: 1E-10, g: 1.0, n: 1.0)), // air optical properties
```

Comment:
I'm seeing an alert here to specify the type (and in several other places), was this intentional to keep the code more concise or align with Python?

Since you have labeled them, I don't think adding the type adds anything except showing which classes are being utilized.

Reply:
What does an "alert" mean, in this case? Are there language features we can't use? Target typed properties remove duplication of info, and I thought perhaps easier to read, in cases like this where you're trying to fit a big idea on the screen.

Reply:
I'm not sure if this is Resharper or SonarLint, sometimes it advises removing the type but in this case is suggests specifying it. Like I said though, I don't think adding it in this case makes it better but that's mainly because you have them labeled, without that I could see where the type name would be useful.

Reply:
I think this is SonarLint and it can be changed to always prefer code without the type so I will resolve this conversation because I think you should leave it the way it is.